src/java.base/share/classes/java/lang/invoke/AbstractValidatingLambdaMetafactory.java

```diff
@@ -53,68 +53,71 @@
      */
     final MethodHandles.Lookup caller;        // The caller's lookup context
     final Class<?> targetClass;               // The class calling the meta-factory via invokedynamic "class X"
-    final MethodType invokedType;             // The type of the invoked method "(CC)II"
-    final Class<?> samBase;                   // The type of the returned instance "interface JJ"
-    final String samMethodName;               // Name of the SAM method "foo"
-    final MethodType samMethodType;           // Type of the SAM method "(Object)Object"
-    final MethodHandle implMethod;            // Raw method handle for the implementation method
-    final MethodType implMethodType;          // Type of the implMethod MethodHandle "(CC,int)String"
+    final MethodType factoryType;             // The type of the invoked method "(CC)II"
+    final Class<?> interfaceClass;            // The type of the returned instance "interface JJ"
+    final String interfaceMethodName;         // Name of the method to implement "foo"
+    final MethodType interfaceMethodType;     // Type of the method to implement "(Object)Object"
+    final MethodHandle implementation;        // Raw method handle for the implementation method
+    final MethodType implMethodType;          // Type of the implementation MethodHandle "(CC,int)String"
     final MethodHandleInfo implInfo;          // Info about the implementation method handle "MethodHandleInfo[5 CC.impl(int)String]"
     final int implKind;                       // Invocation kind for implementation "5"=invokevirtual
     final boolean implIsInstanceMethod;       // Is the implementation an instance method "true"
     final Class<?> implClass;                 // Class for referencing the implementation method "class CC"
-    final MethodType instantiatedMethodType;  // Instantiated erased functional interface method type "(Integer)Object"
+    final MethodType dynamicMethodType;       // Dynamically checked method type "(Integer)Object"
     final boolean isSerializable;             // Should the returned instance be serializable
-    final Class<?>[] markerInterfaces;        // Additional marker interfaces to be implemented
-    final MethodType[] additionalBridges;     // Signatures of additional methods to bridge
+    final Class<?>[] interfaces;              // Additional interfaces to be implemented
+    final MethodType[] bridges;               // Signatures of additional methods to bridge
```

Comment:
If you are removing ACC_BRIDGE from additional generated methods, then perhaps this parameter name could also be changed? `altMethods` (as alternative methods perhaps?)